.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
will this find our `docker/*/Dockerfile`s?

Reply:
Good catch, I'll fix that. Though despite being in documentation I've never actually seen Dependabot suggest a docker upgrade.